src/renderer/model/lobby.ts

```diff
@@ -0,0 +1,41 @@
+// SPDX-FileCopyrightText: 2025 The BAR Lobby Authors
+//
+// SPDX-License-Identifier: MIT
+
+import { UnixTime, StartBox, UserId } from "tachyon-protocol/types";
+//import { Startbox } from "@main/content/maps/map-metadata";
+//import { lobbyPlayer } from "@renderer/model/lobbyPlayer";
+
+// With changes this is now just the same as the tachyon type for LobbyJoinOkResponseData | LobbyCreateOkResponseData
+export type Lobby = {
+    id: string;
+    name: string;
+    mapName: string;
+    engineVersion: string;
+    gameVersion: string;
+    playerCount: number;
+    maxPlayerCount: number;
+    allyTeams?: {
```

Comment:
is there good reason for this to be optional property? similar members etc below

Reply:
Both properties are duplicating the response data for create/join in the protocol. I’ll have to look of I made an error, or if there are reasons for these to be absent. I can’t immediately think of a reason why. A lobby should have at least 1 allyteam with 1 or more team per allyteam.

Reply:
I see that I used the LobbyUpdatedEventData as the base, which has optional properties everywhere because of the patch used.  I'm updating it to the latest tachyon spec anyway, so I'll fix this too.